build/Jenkinsfile.pr

```diff
@@ -124,15 +124,22 @@ pipeline {
 
                 withDockerContainer(args: "-u root --privileged --net ${COMPOSE_PROJECT_NAME}_mm-test -v ${WORKSPACE}/src:/go/src/", image: 'mattermost/mattermost-build-server:dec-7-2018') {
                     ansiColor('xterm') {
-                    sh """
+                        sh """
```

Comment:
Unexpected indentation?

Reply:
Nah, never mind, I read this read -- this is /fixing/ the bad indentation that was there.